src/index.js

```diff
@@ -0,0 +1,85 @@
+/*eslint-disable*/
+import _ from 'lodash'; /*eslint-disable*/
+import './style.css';
+import { Functionality } from './modules/functionality';
+const taskContainer = document.getElementById('first-section');
+const submitButton = document.getElementById('for-button');
+const clearAllDone = document.getElementById('clear-all');
+let editButtonStats = false;
+window.onload = function windowReady() {
+  Functionality.displayTask();
+  submitButton.onclick = function () {
+    Functionality.newTask();
+  };
+
+taskContainer.addEventListener('click', (e) => {
+  if (e.target !== null && e.target !== 'NaN' && e.target !== '') {
+    if (e.target.className === 'checkbox-class') {
+      const ids = e.target.id.replace('checkbox-', '');
+      let description = document.getElementById('d' + ids);
+      let data = Functionality.getAllTasks();
```

Comment:
Some Variable names are not descriptive enough to indicate their purpose, kindly consider that

Reply:
Done.👍🏿